src/BlockTypes/AbstractProductGrid.php

```diff
@@ -357,7 +355,7 @@ public function render_product( $id ) {
 	 * @return string
 	 */
 	protected function get_image_html( $product ) {
-		return '<div class="wc-block-grid__product-image">' . $product->get_image( 'woocommerce_thumbnail' ) . '</div>';
+		return '<div class="wc-block-grid__product-image"><a href="' . esc_url( $product->get_permalink() ) . '">' . $product->get_image( 'woocommerce_thumbnail' ) . '</a></div>';
```

Comment:
This would be a breaking change on the `woocommerce_blocks_product_grid_item_html` filter. The link was previously filterable.

Reply:
Ah, good catch! I will work on a solution if the discussion below is addressed.